src/connector/src/source/datagen/source/reader.rs

```diff
@@ -90,21 +89,20 @@ impl SplitReader for DatagenSplitReader {
 
         // 'fields.f_random_str.length'='10'
         // )
-
+        let mut fields_vec = vec![];
```

Comment:
Previously, it is a hashmap because JSON parser uses the column name to parse, so the order of column does not matter during data generation.